src/Compilers/CSharp/Portable/Symbols/Source/SourceFieldSymbol.cs

```diff
@@ -499,11 +499,15 @@ internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, r
 
             if (type.ContainsNullableReferenceTypes())
```

Comment:
Not related to this PR: it seems `ContainsNullableReferenceTypes` is implemented by manually visiting type symbols. Is it possible to use `VisitType` like `ContainsDynamic` and `ContainsTupleNames`?

Reply:
Added a `// PROTOTYPE` comment to `TypeSymbol.ContainsNullableReferenceTypes`.